source/db/zh-CN-projects.json

```diff
@@ -1287,32 +1289,7 @@
   },
   {
     "development_stage": "released",
-    "description": "私密的 XMPP 服务，运行在 DuckDuckGo 社区平台。&mdash;包含使用说明。",
```

Comment:
@strugee Not my fault here.
https://github.com/nylira/prism-break/commit/3a748b1741e85d1f9968c96c2c6aa584d2eab557